interpreter/cling/lib/Interpreter/Interpreter.cpp

```diff
@@ -666,6 +695,62 @@ namespace cling {
   }
 
   Interpreter::CompilationResult
+  Interpreter::codeComplete(const std::string& line, size_t& cursor,
+                            std::vector<std::string>& completions) const {
+
+    const char * const argV = "cling";
+    std::string llvmDir = getCI()->getHeaderSearchOpts().ResourceDir;
+    // Remove the extra 3 directory names "/lib/clang/3.9.0"
+    StringRef parentDir = llvm::sys::path::parent_path(
+                    llvm::sys::path::parent_path(
+                    llvm::sys::path::parent_path(llvmDir)));
+
+    printf("%s\n", parentDir);
```

Comment:
It does not strip them out:
root [1] myV/home/biancacr/work2/build-completion/etc/cling/lib/clang/3.9.0
ERROR in cling::CIFactory::createCI():
  resource directory /home/biancacr/work2/build-completion/etc/cling/lib/clang/3.9.0/lib/clang/3.9.0 not found!
Am I doing something wrong?